src/heretic/mn_menu.c

```diff
@@ -392,6 +393,7 @@ static Menu_t Crispness1Menu = {
 static MenuItem_t Crispness2Items[] = {
     {ITT_LRFUNC2, "MONO SFX:", CrispySoundMono, 0, MENU_NONE},
     {ITT_LRFUNC2, "SOUND CHANNELS:", CrispySndChannels, 0, MENU_NONE},
+    {ITT_LRFUNC2, "LEVEL WEAPON SOUNDS:", CrispyLvlWpnSnd, 0, MENU_NONE},
```

Comment:
"Level" is misleading here, e.g. this word is used in a completely different context just one menu item below. How about "Adjust weapon sound loudness"?